data/debug-cli/docker_debug.yaml

```diff
@@ -0,0 +1,322 @@
+command: docker debug
+short: Get a shell into any container or image. Replacement for `docker exec`.
+long: |-
+  > **Note**
+  >
+  > Docker Debug requires a [Pro, Team, or Business subcription](/subscription/details/).
+  > You must [sign in](/desktop/get-started/) to use this command.
+
+  Docker Debug helps you follow best practices by keeping your images small and secure. 
+  With Docker Debug, you can debug your images while they contain the bare minimum to run your application.
+  It does this by letting you create and work with slim images or containers that are often difficult to debug because all tools have been removed. 
+  For example, while typical debug approaches like `docker exec -it my-app bash` may not work on a slim container, `docker debug` will work.
```

Comment:
**suggestion**: Maybe we should lead this paragraph with "Docker Debug is a CLI command that..."